support-e2e/tests/landingPages.test.ts

```diff
@@ -18,7 +18,9 @@ test.describe('Paper product page', () => {
 		await page.goto(pageUrl);
 
 		await expect(page.locator('id=qa-paper-subscriptions')).toBeVisible();
-		await expect(page).toHaveURL(/\/uk\/subscribe\/paper/);
+		await expect(
+			page.getByRole('heading', { name: 'Newspaper subscription' }).first(),
+		).toBeVisible();
```

Comment:
Just checking, does this need a corresponding timeout?

Reply:
The massive timeout was waiting for `support-workers` to make the payment. These are just landing pages, so should return in a timely fashion.

(And they didn't have them previously in the current implementation)